src/test/determinism/determinism1_compare.cmake

```diff
@@ -5,8 +5,9 @@ macro(EXEC_DIFF_CHECK FILE1 FILE2)
     endif()
 endmacro()
 foreach(LOOPIDX RANGE 1 10)
-	exec_diff_check(
-		${CMAKE_BINARY_DIR}/determinism1a.shadow.data/hosts/testnode${LOOPIDX}/testnode${LOOPIDX}.testdeterminism.1000.stdout
-		${CMAKE_BINARY_DIR}/determinism1b.shadow.data/hosts/testnode${LOOPIDX}/testnode${LOOPIDX}.testdeterminism.1000.stdout
-	)
+        exec_diff_check(
+            ${CMAKE_BINARY_DIR}/determinism1a-shadow-${METHOD}.data/hosts/testnode${LOOPIDX}/testnode${LOOPIDX}.testdeterminism.1000.stdout
+            ${CMAKE_BINARY_DIR}/determinism1b-shadow-${METHOD}.data/hosts/testnode${LOOPIDX}/testnode${LOOPIDX}.testdeterminism.1000.stdout
+        )
```

Comment:
Check indentation (8 spaces instead of 4?).

Reply:
done